### ISSUE TITLE
KNP version cannot be determined from the git

### ISSUE THREAD

**Issue by maxkost** [KasperskyLab/knp#3]
An error occurs when trying to build from sources:
```
-- Determining version from Git...
CMake Error at cmake/version.cmake:49 (string):
  string sub-command FIND requires 3 or 4 parameters.
Call Stack (most recent call first):
  cmake/version.cmake:59 (determine_version_with_git)
  CMakeLists.txt:31 (knp_get_version)


CMake Error at CMakeLists.txt:34 (message):
  Kaspersky Neuromorphic Platform version is mandatory for build, but it
  cannot be determined.


-- Configuring incomplete, errors occurred!
  ```

**Comment by maxkost** [comment]
@artiomn, I suggested my fix in the PR #4.

**Comment by artiomn** [comment]
Thank you, PR approved.
